### PR TITLE
[#211] Run GrumPHP used in the pre-commit hook in a Docker container

### LINKED ISSUE #211: Run GrumPHP used in the pre-commit hook in a Docker container
In order to prevent installing PHP and XDebug in our host machine to run the pre-commit hook powered by GrumPHP, it could be a good solution running it in a Docker container.

### PR DESCRIPTION
**Pull request by xserrat**
Closes #211

### CODE REVIEW

File: grumphp.yml [hunk 1]
```diff
@@ -1,4 +1,6 @@
 grumphp:
+    git_hook_variables:
+        EXEC_GRUMPHP_COMMAND: 'docker run -e "TERM=xterm-256color"  --tty=true --rm -v $PWD:/code:cached pheature/pheature-php-dev:latest'
```
Comment (kpicaza) [inline grumphp.yml]: Hey @xserrat I added a commit with a colored output example

Comment (xserrat) [reply grumphp.yml]: Oh!! Perfect 👏🏽  🚀

Comment (xserrat) [reply grumphp.yml]: I've just added also to the rest of the packages!! Thanks @kpicaza!!

**Review by kpicaza — APPROVED** [review]
:ship::ship::ship:

**Pull request merged**